src/project.js

```diff
@@ -0,0 +1,91 @@
+const projects = [];
```

Comment:
## ADD a default project
- [ ] According to the project's requirements, there should exist a `default` project on the initial page load. Currently, we don't have this in our project, let's add it.
#### ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/projects_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/todo-list)).